Beta_Integrations/integration-TrendMicroDDA.yml

```diff
@@ -2,7 +2,7 @@ commonfields:
   id: Trend Micro Deep Discovery Analyzer Beta
   version: -1
 name: Trend Micro Deep Discovery Analyzer Beta
-display: Trend Micro Deep Discovery Analyzer (Beta)
+display: Trend Micro Deep Discovery Analyzer
```

Comment:
Why Beta removed? because there is `beta: true`

Reply:
I get \<file-path\> should NOT contain the substring "beta" in a new beta integration.